ccd-definition/ComplexTypes/PayingMoneyAsAResultOfCourtOrder.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "ID": "PayingMoneyAsAResultOfCourtOrder",
+    "ListElementCode": "payingMoneyDetailsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Is ${respondent1.partyName} paying money as a result of any court order? \n",
```

Comment:
The label doesn't start with ## as far as I can see. Or are them special characters for ccd?
Also, instead of using a Label element to write the label for the next field, why not to use the next field's ElementLabel attribute?

Reply:
remove ## from "ElementLabel" 
i am nor sure using about nextfiels's Elemeent Label

Reply:
have discussed this with Dario and he is okay if I resolve this conversation